include/openenclave/internal/load.h

```diff
@@ -105,7 +105,10 @@ struct _oe_enclave_image
         oe_enclave_t* enclave,
         uint64_t* vaddr);
 
-    oe_result_t (*patch)(oe_enclave_image_t* image, size_t enclave_size);
+    oe_result_t (*patch)(
```

Comment:
Since this API is changed in line 110 to be SGX-specific instead of TEE-agnostic, the function needs to be renamed to start with sgx, like line 113 already does

Reply:
renamed to sgx_patch